app/views/shared/_flash.html.erb

```diff
@@ -0,0 +1,3 @@
+<% flash.each do |key, value| %>
```

Comment:
`<% flash.each do |key, value| %>
  <div class="<%= flash_class(key) %>" role="alert">
    <%= value %>
    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
      <span aria-hidden="true">&times;</span>
    </button>
  </div>
<% end %>`

Reply:
We should also consider case when value is an array

  <% flash.each do |key, value| %>
    <div class="<%= flash_class(key) %>" role="alert">
      <a class="close" data-dismiss="alert" href="#_">&times;</a>
      <% if value.is_a?(Array) %>
        <ul>
          <% value.each do |val| %>
            <li><%= val %></li>
          <% end %>
        </ul>
      <% else %>
        <%= value %>
      <% end %>
    </div>
  <% end %>